assemblies/platform/pentaho-cdf-dd/src/main/resources/resources/base/properties/SchemaColumnTypes.xml

```diff
@@ -11,5 +11,13 @@
     <Version>1.0</Version>
   </Header>
 
-  <Values source='Dashboards.listAddIns("Table","colType")'/>
+  <!-- Solr's field types -->
```

Comment:
You mean CDA field types? The reason is that, you either give the properties a name like `solrColumnTypes` or, as I thought, having a general name such as `schemaColumnTypes` it should apply to "all" data sources. Maybe it's just best to name this either `solrColumnTypes` or `cdaColumnTypes`. Against the latter option, is the fact that there already exist some CDA properties with similar names: `Cdacolumns` and `Cdacalculatedcolumns`.